_layouts/project_extended.html

```diff
@@ -0,0 +1,92 @@
+---
+layout: page
+block: project
+---
+
+<section class="project__image">
+    <img src="{{ page.image }}">
+</section>
+
+<section>
+    {% include centerheading.html title="Project Overview: " %}
+    
+    <div class="row">
+        <div class="col-xs-12 col-md-6">
+	<h5><b>Goal:</b></h5>
+        </div>
+        <div class="col-xs-12 col-md-6">
+	
+        </div>
+    </div>
+    
+    <div class="row2">
+        <p class="about__description col-xs-12 col-md-6">
+ The goal of <a href="https://www.indyautonomouschallenge.com">Indy Autonomus Challenge</a> is to implement a software stack capable of operating a Formula One style car in a head to head competition against teams all over the world in a setting similar to the Indy 500. The final race is scheduled for October 23, 2021 at the Indiapolis Motor Speedway, however preceding that there are multiple qualifing rounds including the writing of a white paper, a demonstartion of basic operation, and competing in a simulated race.
+        <div class="col-xs-12 col-md-6">
+           <a href="https://indyautonomouschallenge.com/timeline"> <img class="about__group-image" src="/assets/images/projects/2020/indy-timeline.jpg"></a>
+        </div>
+    </div>
+    
+    <br><br>
+
+    <a href="https://static1.squarespace.com/static/5da73021d0636f4ec706fa0a/t/5e604e57e4fc8938f7a8ed33/1583369815480/University+of+Pittsburgh+White+Paper.pdf">Read Our Whitepaper!</a>
+
+    <br><br>
+
+    {% include centerheading.html title='Project Leads: ' %}
+    <div class="row">
+        <div class="about__person col-xs-12 col-sm-6 col-lg-4">
+            <img src="/assets/images/about/current/nayana-suvarna.jpg">
+            <div class="about__person-title">Nayana Suvarna</div>
+            <div class="about__person-position">Project Co-Lead</div>
+        </div>
+        <div class="about__person col-xs-12 col-sm-6 col-lg-4">
+            <img src="/assets/images/about/current/prem-bharatia.jpg">
+            <div class="about__person-title">Prem Bharatia</div>
+            <div class="about__person-position">Project Co-Lead</div>
+        </div>
+        <div class="about__person col-xs-12 col-sm-6 col-lg-4">
+            <img src="/assets/images/about/alumni/officer-andrew.jpg">
+            <div class="about__person-title">Andrew Saba</div>
+            <div class="about__person-position">Project Mentor</div>
+        </div>
+        <div class="about__person col-xs-12 col-sm-6 col-lg-4">
+            <img src="/assets/images/projects/2020/mentors/josh-spisak.jpg">
```

Comment:
We need to resize this image or something to keep it more consistent

Reply:
Recommended that source image be cropped to a 1:1 (square) aspect ratio. Easiest way to guarantee consistency.

Reply:
resolved